tests/ha/filesystem.pm

```diff
@@ -86,27 +86,40 @@ sub run {
 
     if (is_node(1)) {
         # Create the Filesystem resource
-        assert_script_run
-"EDITOR=\"sed -ie '\$ a primitive $fs_rsc ocf:heartbeat:Filesystem params device='$fs_lun' directory='/srv/$fs_rsc' fstype='$fs_type''\" crm configure edit", $default_timeout;
-
+        my $clean_flag = undef;
+        my $edit_crm_config_script = "#!/bin/sh
+EDITOR='sed -ie \"\$ a primitive $fs_rsc ocf:heartbeat:Filesystem params device=\'$fs_lun\' directory=\'/srv/$fs_rsc\' fstype=\'$fs_type\'\"' crm configure edit
+";
         # Only OCFS2 can be cloned
         if ($fs_type eq 'ocfs2') {
-            assert_script_run "EDITOR=\"sed -ie 's/^\\(group base-group.*\\)/\\1 $fs_rsc/'\" crm configure edit", $default_timeout;
+            $edit_crm_config_script .= "
+EDITOR='sed -ie \"s/^\\(group base-group.*\\)/\\1 $fs_rsc/\"' crm configure edit
+";
```

Comment:
Suggestion about formatting:
```
            $edit_crm_config_script .= "\nEDITOR='sed -ie \"s/^\\(group base-group.*\\)/\\1 $fs_rsc/\"' crm configure edit || exit 1\n";
```

Reply:
Oh, if keeping the string in 3 lines, then no need to add `\n`

Reply:
yes, I updated code